apps/ledger-live-desktop/src/renderer/families/mina/MemoField.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useCallback } from "react";
+import { getAccountBridge } from "@ledgerhq/live-common/bridge/index";
+import invariant from "invariant";
+import { Account } from "@ledgerhq/types-live";
+import { Transaction, TransactionStatus } from "@ledgerhq/live-common/families/mina/types";
+import { useTranslation } from "react-i18next";
+import MemoTagField from "~/newArch/features/MemoTag/components/MemoTagField";
+
+const MemoField = ({
+  onChange,
+  account,
+  transaction,
+  status,
+}: {
+  onChange: (a: Transaction) => void;
+  account: Account;
+  transaction: Transaction;
+  status: TransactionStatus;
+}) => {
+  invariant(transaction.family === "mina", "Memo: Mina family expected");
+
+  const { t } = useTranslation();
+
+  const bridge = getAccountBridge(account);
+
+  const onMemoFieldChange = useCallback(
+    (value: string) => {
+      if (value !== "") onChange(bridge.updateTransaction(transaction, { memo: value }));
```

Comment:
Why do we need to split cases here ? (undefined vs value)

Reply:
This is because memo was defined optional in the beginning and we stuck with that. Is this a problem?